### ISSUE TITLE
Image Configurator does not wrk

### ISSUE THREAD

**Issue by noelrodriguez80** [TobiZog/cinnamon-dynamic-wallpaper#9]
The Button does not do anything.. I press/click on in and it just blinks.
Nothing happens.

I reinstalled twice with same restult

**Comment by TobiZog** [comment]
Can you please open "Melange" after you clicked on the preferences button in the Extensions window? You can open it with the shortkey "Win/Super" + "L"

**Comment by noelrodriguez80** [comment]
Sorry for the delay, Xmas got on the way.

I am not sure if I am doing this correctly.
I opened the extensions  (see "DynaWall002.png", I see two tabs, Manage and
Download but no 'Preferences' in either window).

Any way I enabled  dynamic wallpaper with the '+' and click on the
configuration button

Then Win/L to get 'melange', the result is on DynaWall001.png

I don't see the Preferences button, I do see a configuration button but
nothing shows on "Melange"
